node/cmd/ccq/permissions.go

```diff
@@ -205,14 +215,27 @@ func parseConfig(byteValue []byte, allowAnything bool) (PermissionsMap, error) {
 		}
 
 		if user.AllowAnything {
-			if !allowAnything {
+			if !config.AllowAnythingSupported {
 				return nil, fmt.Errorf(`UserName "%s" has "allowAnything" specified when the feature is not enabled`, user.UserName)
 			}
 			if len(user.AllowedCalls) != 0 {
 				return nil, fmt.Errorf(`UserName "%s" has "allowedCalls" specified with "allowAnything", which is not allowed`, user.UserName)
 			}
 		}
 
+		var rateLimiter *rate.Limiter
+		rateLimit := config.DefaultRateLimit
+		if user.RateLimit != nil {
+			rateLimit = *user.RateLimit
+		}
+		if rateLimit != 0 {
+			burstSize := config.DefaultBurstSize
```

Comment:
It might be good to have `config.DefaultBurstSize` default to 1 instead of 0, otherwise if only `DefaultRateLimit` is set, all requests will be blocked (if I understand `rate` correctly)

Reply:
Made it default burst size to one and not allow it to be set to zero.